tensorflow/python/compiler/tensorrt/__init__.py

```diff
@@ -19,5 +19,74 @@
 from __future__ import print_function
 
 # pylint: disable=unused-import,line-too-long
+from tensorflow.compiler.tf2tensorrt import _pywrap_py_utils
 from tensorflow.python.compiler.tensorrt import trt_convert as trt
+from tensorflow.python.compiler.tensorrt import utils as trt_utils
+from tensorflow.python.platform import tf_logging as logging
 # pylint: enable=unused-import,line-too-long
+
+
+if not _pywrap_py_utils.is_tensorrt_enabled():
+  """Check that TensorFlow has been built with TensorRT support enabled.
+  Raises:
+    RuntimeError: if the TensorFlow has not be built with TensorRT support.
+  """
```

Comment:
Is this just a comment? Why do we even need this comment to repeat what is in the code? Let's delete this few lines.